### ISSUE TITLE
Should throw an error if invalid XML Policy root element.

### ISSUE THREAD

**Issue by kurtkanaskie** [apigee/apigeelint#278]
I mean, if I somehow create a policy with an unknown root element (e.g. CORS-v1), Apigelint should throw an error.

For example:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<CORS-v1 continueOnError="false" enabled="true" name="CORS-1">
    <DisplayName>CORS-1</DisplayName>
    <AllowOrigins>{request.header.origin}</AllowOrigins>
    <AllowMethods>GET, PUT, POST, DELETE</AllowMethods>
    <AllowHeaders>origin, x-requested-with, accept, content-type</AllowHeaders>
    <ExposeHeaders>*</ExposeHeaders>
    <MaxAge>3628800</MaxAge>
    <AllowCredentials>false</AllowCredentials>
    <GeneratePreflightResponse>true</GeneratePreflightResponse>
    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
</CORS-v1>
```
